.gitignore

```diff
@@ -1,3 +1,8 @@
 /build
 /lib/resty/auto-ssl/vendor/*
 /t/run
+/t/build
+/t/tmp
+/t/logs
+/t/vendor
+/t/servroot*
```

Comment:
As I [noted](https://github.com/GUI/lua-resty-auto-ssl/pull/67#issuecomment-300664140) I don't think these gitignore changes should be needed.